### DISCUSSION TITLE
Unable to add error_map coordinate for properties of nested objects

### DISCUSSION THREAD

**Discussion by ErikKoinberg** [bhch/django-jsonform#151] [Q&A] [answered]
I have a simple object with a basic property, the schema looks like this: 

`schema = {
    'type': 'object',
    'properties': {
        'test': {
            'type': 'object',
            'properties': {
                'testProp': {'type': 'string'}
            }
        }
    }
}`

I am able to add an error message to 'test' using: 
`error_map['test'] = 'Some error'`

but when trying to set an error on the nested property, nothing shows. I am doing it like this: 
`error_map['test-testProp'] = 'Some nested error'`

Another question, does the coordinate system handle keys with spaces inside of them?

**Comment by bhch** [comment, marked as answer]
`'test-testProp'` is not a valid coordinate. We don't use hyphens (`-`) to join nested coordinates. Instead, we use the section sign (`§`). This is due to #69.

So the valid coordinate would be `'test§testProp'`.

We've got a [`join_coords`](https://django-jsonform.readthedocs.io/en/latest/utils.html#join-coords) utility function to chain the coordinates. You should use that and it will take care of using the appropriate symbol to use for joining.

> Another question, does the coordinate system handle keys with spaces inside of them?

Yes.
